### PR TITLE
Updated text about setting docroot to pub.

### LINKED ISSUE #3540: Installation on /pub
# Topic clarification/correction request

## Topic Link

https://devdocs.magento.com/guides/v2.3/install-gde/tutorials/change-docroot-to-pub.html

## Description

The page says:

> If you installed Magento in Apache’s default docroot /var/www/html, the Magento file system is vulnerable because it’s accessible from a browser. This topic describes how to change the Apache docroot on an existing Magento instance to serve files from the Magento pub/ directory, which is more secure.
> 
> Serving files from the pub/ directory prevents site visitors from accessing the Web Setup Wizard and other sensitive areas of the Magento file system from a browser.

The problem is that this is the only way to install magento on /pub there is not another way. We can't install directly from /pub we get a loop error (https://github.com/magento/magento2/issues/16538).

The documentation misguide the user, the sentence "If you installed Magento in Apache’s default docroot /var/www/html" lets the user understand that there are multiple ways to do it and it is not correct.

## Possible solutions

The page seems a tutorial on how to fix something that you have missed, you forgot to put it on /pub you can fix it following this how-to. It is wrong, you should set up the page in a different way, the page should be part of the installation devdocs, a possible final step get Magento works on /pub after the standard installation, to improve the security.

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

Clarifies the discussion around setting the webroot to /pub on Apache.
Fixes #3540

### CODE REVIEW

File: guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md [hunk 1]
```diff
@@ -3,19 +3,29 @@ group: installation-guide
...
-If you're accustomed to using the Web Setup Wizard during development, be aware that you won't be able to access it when serving files from the `pub/` directory.
+If you're accustomed to using the Web Setup Wizard during development, be aware that you will not be able to access it when serving files from the `pub/` directory.
 
 ## A note about nginx
 
-If you're using [nginx]({{ page.baseurl }}/install-gde/prereq/nginx.html) and the [`nginx.conf.sample`]({{ site.mage2200url }}nginx.conf.sample){:target="_blank"} file included in the Magento installation directory, you're probably already serving files from the `pub/` directory.
-
-The sample configuration overrides your server's docroot settings to serve files from Magento's `pub/` directory; assuming you've referenced the `nginx.conf.sample` in the server block that defines your site. For example, see the last line in the following configuration:
+If you are using [nginx]({{ page.baseurl }}/install-gde/prereq/nginx.html) and the [`nginx.conf.sample`]({{ site.mage2200url }}nginx.conf.sample){:target="_blank"} file included in the Magento installation directory, you are probably already serving files from the `pub/` directory.
```
Comment (lorikrell) [inline guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md]: Don't need the blank anymore for links, just a slight edit.

```suggestion
If you are using [nginx]({{ page.baseurl }}/install-gde/prereq/nginx.html) and the [`nginx.conf.sample`]({{ site.mage2200url }}nginx.conf.sample) file included in the Magento installation directory, you are probably already serving files from the `pub/` directory.
```

File: guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md [hunk 2]
```diff
@@ -3,19 +3,29 @@ group: installation-guide
...
 
+The sample configuration overrides your server's docroot settings to serve files from Magento's `pub/` directory, assuming you have referenced the `nginx.conf.sample` in the server block that defines your site. For example, see the last line in the following configuration:
```
Comment (lorikrell) [inline guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md]: Just a suggested edit:

```suggestion
When used in your server block that defines your site, the `nginx.conf.sample` configuration overrides your server's docroot settings to serve files from Magento's `pub/` directory. For example, see the last line in the following configuration:
```

File: guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md [hunk 3]
```diff
@@ -29,10 +39,10 @@ The sample configuration overrides your server's docroot settings to serve files
               set $MAGE_ROOT /var/www/html/magento2ce;
               include /var/www/html/magento2ce/nginx.conf.sample;
     }
-
+```
```
Comment (lorikrell) [inline guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md]: Just make sure there is an empty line between the code ticks and Before you begin section.

File: guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md]: Thanks.

File: guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md]: THanks.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**